nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>Mention that this is equivalent to having these two quarto cell directives</p><p> </p><p>Allows you to hide the cell input</p><p>#| echo: false</p><p> </p><p>Allows you to hide the cell output</p><p>#| output: false</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>